charts/cluster-autoscaler/Chart.yaml

```diff
@@ -17,4 +17,4 @@ name: cluster-autoscaler
 sources:
   - https://github.com/kubernetes/autoscaler/tree/master/cluster-autoscaler
 type: application
-version: 9.16.2
+version: 9.16.3
```

Comment:
This will need rebased anyway as I've just merged a different PR, but could you please make this a minor bump rather than patch as it's introducing new functionality?

Other than that this looks good to me, thanks for the PR!

Reply:
@gjtempleton 

Thanks for your review and comment. 

>minor bump rather than patch as it's introducing new functionality

I agree, so I bumped minor version in Chart.yaml.